lib/order.rb

```diff
@@ -0,0 +1,56 @@
+class Order
+    attr_reader :id, :products, :customer, :fulfillment_status
+
+    FULFILLMENT_STATUS = [:paid, :processing, :shipped, :complete, :pending]
+    
+    def initialized(id, customer, products, fulfillment_status = :pending)
```

Comment:
You spelled `initialize` wrong.

```suggestion
    def initialize(id, customer, products, fulfillment_status = :pending)
```